ci/ci_integration.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+rake test:install-all
```

Comment:
I believe default plugin would be sufficient and will go around some issue with have with uninstalling plugins that has cross dependencies.